tests/consensus_test.rs

```diff
@@ -239,6 +268,7 @@ impl TestNetwork {
     pub fn stop(&self) {
         for node in self.nodes.iter() {
             node.stop();
+            node.db.destroy().unwrap();
```

Comment:
Does this get called correctly if tests fail/panic?

Reply:
Ah no it doesn't. I implemented the `Drop` trait on `NodeForTest` to handle cleanup. Might be better to impl this closer to the source (e.g. on the rocksdb type) but this seems like it does the job for now.